cmd/mesh-bootstrap/main.go

```diff
@@ -290,18 +286,30 @@ func main() {
 		LogBandwidthUsageStats: true,
 	})
 
-	// Protect each other bootstrap peer via the connection manager so that we
-	// maintain an active connection to them. Also prevent other bootstrap nodes
-	// from being banned.
-	bootstrapAddrInfos, err := p2p.BootstrapListToAddrInfos(bootstrapList)
-	if err != nil {
-		log.WithField("error", err).Fatal("could not parse bootstrap list")
-	}
-	for _, addrInfo := range bootstrapAddrInfos {
-		connManager.Protect(addrInfo.ID, "bootstrap-peer")
-		for _, addr := range addrInfo.Addrs {
-			_ = banner.ProtectIP(addr)
+	bootstrapList := p2p.DefaultBootstrapList
```

Comment:
nit: If `bootstrapList` isn't used anywhere else we can scope it inside of the `if config.UseBootstrapList` block.

Reply:
👍